terraform-module/modules/frontend-spa-cdn/main.tf

```
@@ -148,6 +142,36 @@ resource "aws_cloudfront_distribution" "this" {
   }
 }
 
+resource "aws_cloudwatch_log_delivery_source" "cloudfront_log_source" {
+  region = "eu-west-1"
+
+  name         = "cloudfront_log_source"
+  log_type     = "ACCESS_LOGS"
+  resource_arn = aws_cloudfront_distribution.this.arn
+}
+
+resource "aws_cloudwatch_log_delivery_destination" "cloudfront_log_destination" {
+  region = "eu-west-1"
+
+  name          = "cloudfront_log_destination"
```

we know destination, which is s3
```suggestion
  name          = "s3"
```